src/main/java/se/sundsvall/citizenassets/api/AssetResource.java

```diff
@@ -50,13 +47,18 @@ public AssetResource(AssetService service) {
 
 	@GetMapping
 	@ApiResponse(responseCode = "200", description = "OK", useReturnTypeSchema = true)
+	@ApiResponse(responseCode = "400", description = "Bad Request", content = @Content(mediaType = APPLICATION_PROBLEM_JSON_VALUE, schema = @Schema(implementation = Problem.class)))
```

Comment:
Any reason for duplicating e.g. the ApiResponse for a 400 at each method instead of having it on the "class"-level?

Reply:
In my opinion, having it on class level seems more clean. This just feels like duplication.

Reply:
Ok, in my opinion it's easier to read all response codes together with the method, but I'll change back.